frontend/js/pages/CardTeam.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
```

Comment:
you can disable this globally on the eslint file config

Reply:
Okay, I'll do this, thanks! :)